### PR TITLE
section list for user profile

### PR DESCRIPTION
**Pull request by lee-cheung**

### CODE REVIEW

File: NativeApp/src/components/User/View/index.js [hunk 1]
```diff
@@ -44,6 +42,17 @@ const UserView = (props) => {
...
                 />
               )}
+              renderSectionHeader={({ section }) => (
+                section.data.length >= 1
+                  ? (<H3 type="bold" style={styles.sectionListHeader}>{section.title}</H3>)
+                  : (
+                    <View>
+                      <H3 type="bold" style={styles.sectionListHeader}>{section.title}</H3>
+                      <H3 style={styles.sectionListNoItems}>Nothing to show</H3>
+                    </View>)
```
Comment (William-Unosquare) [inline NativeApp/src/components/User/View/index.js]: Is there anywhere else you can display the "nothing to show"? Feels weird to be sticking a div and 2 h3's inside 1 header.

File: NativeApp/src/components/User/container.js [hunk 2]
```diff
@@ -46,6 +47,25 @@ export default Container => class extends Component {
     this.sub.remove();
   }
 
+  sortingEvents = (events) => {
+    let eventArray = [];
+    let approvalArray = [];
+    let awaitApprovalArray = [];
+
+    approvalArray.push(events.filter(event => event.eventStatus.description === 'Approved'));
+    awaitApprovalArray.push(events.filter(event => event.eventStatus.description === 'Awaiting approval'));
```
Comment (William-Unosquare) [inline NativeApp/src/components/User/container.js]: I'd call these something more like `approvedEvents` and `pendingEvents`. If "awaitApproval" is the terminology you've used elsewhere in the app then ignore me. :)

**Review by William-Unosquare — CHANGES_REQUESTED** [review]
Question & a wee possible change.

File: NativeApp/src/components/User/View/index.js (hunk 1 above)
Comment (bgribz) [reply NativeApp/src/components/User/View/index.js]: I kinda think that multiline ternaries like these are a bit untidy. Another approach would be to move the logic to a function that will return either component, or ideally do this kind of logic in a container that will pass the correct component through on props.

**Review by bgribz — CHANGES_REQUESTED** [review]
Just a small suggestion

File: NativeApp/src/components/User/View/index.js (hunk 1 above)
Comment (lee-cheung) [reply NativeApp/src/components/User/View/index.js]: i have made changes as per you suggestions. please could you check it out? thanks :)

**Review by bgribz — APPROVED** [review]

**Review by William-Unosquare — APPROVED** [review]
Looks good

**Pull request merged**